client/src/Clusters/RecommendedCategories/recommended_categories.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import CardContainer from './card_container.js';
+import { cluster } from './mock_data.js';
+import '../styles.css';
+
+class RecommendedCategories extends Component {
+    render() {
+        const makeCard = item => (
+            <CardContainer
+                key={item.id}
+                category={item.category}
+                url={item.url}
+            />
+        );
+
+        return (
+            <div className="category-section">
+                <div className="category-section-header">
+                    <h6>Didn't find what you're looking for?</h6>
```

Comment:
Please move the title and subtitle into the mock cluster. All data should be coming from the back end. The front end should only be concerned with data placement and styling